### PR TITLE
Minor tweaks to PIFBuilder and PIF types

### PR DESCRIPTION
**Pull request by hartbit**
Minor reformatting and NFC I did while reading over and studying the `XCBuildSupport` target.

### CODE REVIEW

File: Sources/XCBuildSupport/PIF.swift [hunk 1]
```diff
@@ -22,26 +22,11 @@ import Foundation
...
-
-            // Encode the projects and their targets.
-            for project in workspace.projects {
-                try container.encode(project)
-                for target in project.targets {
-                    try container.encode(target)
-                }
-            }
-        }
+    public struct Document {
```
Comment (aciidb0mb3r) [inline Sources/XCBuildSupport/PIF.swift]: I would rather call it PIFObject than Document to keep things familiar.

Comment (hartbit) [reply Sources/XCBuildSupport/PIF.swift]: It's already in the `PIF` enum, and all other types in the `PIF` enum don't have a `PIF` prefix.

Comment (aciidb0mb3r) [reply Sources/XCBuildSupport/PIF.swift]: Maybe call it TopLevelObject then ?

Comment (hartbit) [reply Sources/XCBuildSupport/PIF.swift]: Sure.

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Pull request merged**